helm-charts/mock-classifier/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
I didn't think we needed a service account, did we? Please correct me if I've misunderstood.

Reply:
we do not, the authentication is done in the pipeline, this was generated automatically during the helm create command I will delete this file